packages/network-console-shared/src/file_io/index.ts

```diff
@@ -1,14 +1,25 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { ICollectionFormat } from './interfaces';
+import { ICollectionFormat, IEnvironmentFormat } from './interfaces';

```

Comment:
is this used?

Reply:
Yes. It's used as an export intrinsically by the compiler. (I think I made a point of this in src/index.ts, but it's required here, too).